inspirehep/modules/workflows/workflows/article.py

```diff
@@ -342,16 +344,7 @@
 
 
 MARK_IF_MATCH_IN_HOLDINGPEN = [
```

Comment:
`STOP_IF_MATCH_IN_HOLDINGPEN`?

Reply:
I didn't want to change name of the workflows, but you may be right to change this to different name. I think that stop is not a good word as it's used for workflows which are stopped permanently and they status is set to COMPLETED.